packages/web/components/Header/Logo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+interface IProps {
+  title: string;
+  src?: string;
+}
+
+const Logo: React.FC<IProps> = React.memo(({ title, src, ...rest }) => {
```

Comment:
Twoje propsy tutaj to atrybuty `img`, więc można użyć ze specjalnego typu `HTMLImageElement`

Reply:
Skoro też chcesz wyodrębnić `rest`, przypuszczam że tam mogłyby przyjść style, to można zrobić tak:
```
interface IProps {
  wrapperCss: InterpolationWithTheme<any> // Nie jestem pewien co do tego typu, pisze do z pamięci
}

const Logo: React.FC<IProps & HTMLImageElement> = = React.memo(({ wrapperCss, ...rest })

```

I wtedy do `img` spreadujesz `rest`, a `wrapperCss` przekazujesz tam gdzie potrzebujesz.

EDIT: zamiast rest możesz użyć nazwy `imgProps`